### PR TITLE
Clarify that URIs shall also be converted

### LINKED ISSUE #3199: convertClass causes broken Modelica-URIs
As originally observed by @mwetter in https://github.com/lbl-srg/modelica-buildings/pull/2960.

Currently, if the convertClass rule of a conversion script is applied, existing references in Modelica-URIs (and basically any other text) to the old type become invalid.

Is this a hole in the conversion rules or can it be seen as tool issue?

Example package:

```mo
package P
  block B
    extends Modelica.Blocks.Interfaces.BlockIcon;
    annotation(Documentation(info="<html>
<p>This will become a broken Modelica-URI (after class conversion) in MSL 4.0.0:
<a href=\"modelica://Modelica.Blocks.Interfaces.BlockIcon\">Modelica.Blocks.Interfaces.BlockIcon</a>.
</p>
</html>"));
  end B;
  annotation(uses(Modelica(version="3.2.1")));
end P;
```

### PR DESCRIPTION
**Pull request by HansOlsson**
Closes #3199

### CODE REVIEW

File: chapters/annotations.tex [hunk 1]
```diff
@@ -1604,6 +1604,7 @@ \subsubsection{Conversion Rules}\label{conversion-rules}
 
 \begin{nonnormative}
 Whenever possible tools should preserve the original style of the model, e.g.\ use of imports.
+Conversions should be applied in all places where classes are used in code, including modelica:// links in documentation annotations.
```
Comment (henrikt-ma) [inline chapters/annotations.tex:1607]: They should be called _Modelica URIs_, right?
```suggestion
Conversions should be applied in all places where classes are used in code, including Modelica URIs  in documentation annotations.
```

**Comment by HansOlsson** [comment]
> Conversions apply to component names too, so maybe say "elements" or "element names" instead of "classes".

Yes, that's a good proposal. I'm also trying to think if there's a better word than 'used' to indicate that we semantically reference the element.

**Review by casella — APPROVED** [review]
LGTM

File: chapters/annotations.tex [hunk 2]
```diff
@@ -1604,6 +1604,7 @@ \subsubsection{Conversion Rules}\label{conversion-rules}
 
 \begin{nonnormative}
 Whenever possible tools should preserve the original style of the model, e.g.\ use of imports.
+Conversions should be applied in all places where classes are used in code, including Modelica URIs (for example, in \lstinline!Documentation! annotations).
```
Comment (HansOlsson) [inline chapters/annotations.tex:1607]:
```suggestion
Conversions should be applied in all places where named element are used as code, including Modelica URIs (for example, in \lstinline!Documentation! annotations).
```
```suggestion
Conversions should be applied in all places where classes are used in code, including Modelica URIs (for example, in \lstinline!Documentation! annotations).
```

Comment (HansOlsson) [inline chapters/annotations.tex:1607]:
```suggestion
Conversions should be applied in all places where named element are used as code, including Modelica URIs (for example, in \lstinline!Documentation! annotations).
```
```suggestion
Conversions should be applied in all places where classes are used in code, including Modelica URIs (for example, in \lstinline!Documentation! annotations).
```

**Pull request merged**